appconfiguration/azure-spring-cloud-appconfiguration-config/azure-spring-cloud-appconfiguration-config-convert-sample/azure-spring-cloud-appconfiguration-config-convert-sample-initial/README.md

```diff
@@ -278,6 +278,10 @@ In this section, you clone a containerized Spring Boot application and test it l
 
 1. You should see the following message displayed: **findOne in User collection get result: testFirstName**
 
+## Deploy to Azure Spring Apps
```

Comment:
Is there a reason we are recommending Azure Spring App over other azure services? This extra doesn't make since for app config as there is no real integration there and we should make it clear that Azure App Configuration doesn't have anything to do with the various configuration option given in Azure Spring Apps.

Reply:
this is the overall devdiv strategy that ASA is designed for spring apps